bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -52,39 +58,85 @@ public boolean allowedInCommit() {
+            reply.println(USER_INVALID_WARNING);
+            return;
+        }
+
+        if (pr.isClosed() && !pr.labelNames().contains("integrated")) {
+            reply.println("`/backport` command can not be used in a closed but not integrated pull request");
             return;
         }
 
         var args = command.args();
         if (args.isBlank()) {
-            showHelp(reply);
+            showHelpInPR(reply);
             return;
         }
 
         var parts = args.split(" ");
-        if (parts.length > 2) {
-            showHelp(reply);
+        boolean argIsValid = parts[0].equals("disable") ? parts.length == 2 || parts.length == 3 : parts.length <= 2;
+        if (!argIsValid) {
+            showHelpInPR(reply);
             return;
         }
 
+        if (parts[0].equals("disable")) {
+            // Remove label
+            var targetRepoName = parts[1];
+            var targetBranchName = parts.length == 3 ? parts[2] : "master";
+            var backportLabel = generateBackportLabel(targetRepoName, targetBranchName);
+            if (pr.labelNames().contains(backportLabel)) {
+                labelsToRemove.add(backportLabel);
+                reply.println("Backport for repo `" + targetRepoName + "` on branch `" + targetBranchName + "` was successfully disabled.");
+            } else {
+                reply.println("Backport for repo `" + targetRepoName + "` on branch `" + targetBranchName + "` was already disabled.");
+            }
+        } else {
+            // Get target repo
+            var targetRepo = getTargetRepo(bot, parts, reply);
+            if (targetRepo == null) {
+                return;
+            }
+            var targetRepoName = targetRepo.name();
+
+            // Get target branch
+            var targetBranch = getTargetBranch(parts, targetRepo, reply);
+            if (targetBranch == null) {
+                return;
+            }
+            var targetBranchName = targetBranch.name();
+
+            // Add label
+            var backportLabel = generateBackportLabel(targetRepoName, targetBranchName);
+            if (pr.labelNames().contains(backportLabel)) {
+                reply.println("Backport for repo `" + targetRepoName + "` on branch `" + targetBranchName + "` has already been enabled.");
+            } else {
+                labelsToAdd.add(backportLabel);
+                reply.println("Backport for repo `" + targetRepoName + "` on branch `" + targetBranchName + "` was successfully enabled and will be performed once this pull request has been integrated.");
+                reply.println("Further instructions will be provided at that time.");
+                reply.println("<!-- " + command.user().username() + " -->");
+            }
+        }
+    }
+
+    private String generateBackportLabel(String targetRepo, String targetBranchName) {
+        return "Backport=" + targetRepo + ":" + targetBranchName;
```

Comment:
I think all of the labels we use for OpenJDK use only lowercase, so I would suggest using `backport=<repo>:<branch>` for the labels.

Reply:
Thanks for the advice!  I will change it to lowercase.